.github/CONTRIBUTING.md

```diff
@@ -73,7 +89,8 @@ If we suggest changes, then:
 If the PR gets too outdated we may ask you to rebase and force push to update the PR:
 
 ```shell
-git rebase master -i
+git fetch upstream
+git rebase upstream/master
 git push origin my-fix-branch -f
```

Comment:
Shouldn't this be upstream/my-fix-branch?

Reply:
No. Origin is your fork, upstream is runelite/runelite

Reply:
Whoops. It's late.